### ISSUE TITLE
Electrum Android: Unable to select coins to spend from

### ISSUE THREAD

**Issue by jbrepogmailcom** [spesmilo/electrum#7790]
Hello. When paying in electrum android, the fee is extremely high, because it select multiple UTXO's. Can I select just one coin like in desktop version?

**Comment by SomberNight** [comment]
No, the Android app (kivy gui) currently does not have an equivalent of the `Coins` tab in Qt.
There is no convenient way of coin control / utxo selection.

As a workaround, you can freeze/unfreeze addresses, to force the coinchooser to use UTXOs from unfrozen addresses, but it's quite tedious to do.

**Comment by nc50lc** [comment]
I think this can be closed since it's a duplicate of: [5661](https://github.com/spesmilo/electrum/issues/5661)
The issue is already answered with a workaround anyways.
